apps/ledger-live-desktop/src/renderer/components/OperationsList/DateCell.tsx

```diff
@@ -16,72 +17,62 @@ const Cell = styled(Box).attrs(() => ({
   width: auto;
   min-width: ${p => (p.compact ? 90 : 120)}px;
 `;
+
 type Props = {
   t: TFunction;
   operation: Operation;
   text?: string;
   compact?: boolean;
   editable?: boolean;
 };
-class DateCell extends PureComponent<Props> {
-  static defaultProps = {
-    withAccount: false,
-  };
 
-  // less than 5 minutes: loading spinner; more than 5 minutes: yellow warning icon
-  pendingLoadingIcon(isOlderThan5Min: boolean) {
-    if (isOlderThan5Min) {
-      return (
-        <Box style={{ verticalAlign: "sub", display: "inline" }}>
-          <WarningSolidMedium size={12} color={"#FFBD42"} />
-        </Box>
-      );
-    } else {
-      return <InfiniteLoader size={12} style={{ verticalAlign: "middle" }} />;
-    }
+const PendingLoadingIcon = ({ displayWarning }: { displayWarning: boolean }): JSX.Element => {
+  if (displayWarning) {
+    return (
+      <Box style={{ verticalAlign: "sub", display: "inline" }}>
+        <WarningSolidMedium size={12} color={"#FFBD42"} />
+      </Box>
+    );
   }
 
-  render() {
-    const { t, operation, compact, text, editable } = this.props;
-    const ellipsis = {
-      display: "block",
-      textOverflow: "ellipsis",
-      overflow: "hidden",
-      whiteSpace: "nowrap" as const,
-    };
-    return (
-      <Cell compact={compact}>
-        <Box ff="Inter|SemiBold" fontSize={3} color="palette.text.shade80" style={ellipsis}>
-          {text ||
-            t(operation.hasFailed ? "operationDetails.failed" : `operation.type.${operation.type}`)}
-        </Box>
-        {editable ? (
-          <Box fontSize={3} color="palette.text.shade80">
-            <Box ff="Inter|SemiBold" fontSize={3} color="palette.text.shade80" style={ellipsis}>
-              {
-                // display pending icon (less than 5 minutes: loading spinner; more than 5 minutes: yellow warning icon)
-                this.pendingLoadingIcon(
-                  new Date().getTime() - operation.date.getTime() >
-                    getEnv("ETHEREUM_STUCK_TRANSACTION_TIMEOUT"),
-                )
-              }
-              <Box
-                style={{
-                  marginLeft: "4px",
-                  verticalAlign: "middle",
-                  display: "inline-block",
-                  fontSize: "10px",
-                }}
-              >
-                {t("operation.type.SENDING") + "..."}
-              </Box>
+  return <InfiniteLoader size={12} style={{ verticalAlign: "middle" }} />;
+};
+
+const DateCell = ({ t, operation, compact, text, editable }: Props): JSX.Element => {
+  const ellipsis = {
+    display: "block",
+    textOverflow: "ellipsis",
+    overflow: "hidden",
+    whiteSpace: "nowrap" as const,
```

Comment:
It is, otherwise you get a type error (`Type 'string' is not assignable to type 'WhiteSpace | undefined'` ) when using the `ellipsis` object as a `style` prop.

The `WhiteSpace` type definition is as follow in [csstype lib](https://github.com/frenic/csstype) used by our `ui` lib
`type Property.WhiteSpace = "pre" | Globals | "normal" | "nowrap" | "-moz-pre-wrap" | "break-spaces" | "pre-line" | "pre-wrap"`

I let you dig locally if you are curious